packages/textlint-tester/src/textlint-tester.ts

```diff
@@ -71,47 +119,109 @@ export class TextLintTester {
         }
     }
 
-    testValidPattern(ruleName: string, rule: TextlintRuleCreator, valid: TesterValid) {
+    testValidPattern(name: string, param: TextlintRuleCreator | TestConfig, valid: TesterValid) {
         const text = typeof valid === "object" ? valid.text : valid;
         const inputPath = typeof valid === "object" ? valid.inputPath : undefined;
-        const options = (typeof valid === "object" && valid.options) || {};
         const ext = typeof valid === "object" && valid.ext !== undefined ? valid.ext : ".md";
         const textlint = new TextLintCore();
-        textlint.setupRules(
-            {
-                [ruleName]: rule
-            },
-            {
-                [ruleName]: options
+        if (isTestConfig(param)) {
+            const rules: any = {};
```

Comment:
Can use `{ [index: string]: TextlintRuleCreator; }` instead of any.

- [Index Signatures · TypeScript Deep Dive](https://basarat.gitbooks.io/typescript/docs/types/index-signatures.html "Index Signatures · TypeScript Deep Dive")

Reply:
An index signatures! Fixed.